packages/slate/src/interfaces/element.js

```diff
@@ -1547,7 +1547,7 @@ class ElementInterface {
     this.assertNode(newParentPath)
 
     const [p, np] = PathUtils.crop(path, newPath)
-    const position = PathUtils.compare(p, np)
+    const position = PathUtils.compareOfSameSize(p, np)
```

Comment:
I think since we already used `crop` that this can also use the new `compare` instead?

Reply:
It can! I'll fix it.